core/pom.xml

```diff
@@ -135,6 +135,12 @@
       <version>${asciidoctorj-pdf.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
No issue to get it but got some hard time requiring it ~1year earlier (in another project) cause of conflicts, wonder if we shouldnt use properties (ie built in in the JVM) by default. Using several entries (attributes.pdf, attributes.html etc... makes it easy enough IMO).

WDYT?